### PR TITLE
feat: adds markdown linting for consistent community document styling

### PR DESCRIPTION
**Pull request by jpower432**
## What Changed?

1. Adds a new workflow for markdown linting
2. Update the proposal workflow for workflow consistency
3. Updates documents with fixes for issues found by the linter

Changes require two approvals per the `GOVERNANCE.md`

### CODE REVIEW

**Review by jflowers — APPROVED** [review]
LGTM

**Review by vikas-agarwal76 — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -61,13 +61,12 @@ We also have a [shared calendar](https://zoom-lfx.platform.linuxfoundation.org/m
...
 
-##### Email
+### Email
 
 Google Group: [oscal-compass](https://groups.google.com/g/oscal-compass)\
 oscal-compass@googlegroups.com
 
-
-##### Chat anytime
+### Chat anytime
```
Comment (degenaro) [inline README.md:69]: Maybe just me, but when I follow link to chat I am prompted to login. Fine. I choose email and get code to enter. Fine. Then I arrive at slack and am prompted to sign in to kubernetes. This is correct?

**Review by degenaro — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (jpower432) [reply README.md:69]: I tried to reproduce, but I am always directed to CNCF Slack.

**Pull request merged**